vllm/config/compilation.py

```diff
@@ -97,19 +97,25 @@ class PassConfig:
 
     This is separate from general `CompilationConfig` so that inductor passes
     don't all have access to full configuration - that would create a cycle as
-    the `PassManager` is set as a property of config."""
+    the `PassManager` is set as a property of config.
```

Comment:
This was actually changed recently; we should still keep pass config separate, but this reason is no longer true. Can be done in follow up cc @ilmarkov